### PR TITLE
regexp-v2: add a test for empty pattern with seqscan

### PR DESCRIPTION
**Pull request by komainu8**
We can't add tests for indexscan and bitmapscan. Because they reject an empty pattern for now.
But we'll accept it in a separated PR. We'll add tests for them in the PR.

### CODE REVIEW

File: sql/regexp/text-array/regexp-v2/empty-string/seqscan.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+);
+
+INSERT INTO memos
+     VALUES (ARRAY['PostgreSQL is a RDBMS',
+                   'Groonga is fast full text search engine',
+                   'PGroonga is a PostgreSQL extension that uses Groonga']);
+
+CREATE INDEX pgrn_content_index
+    ON memos
+ USING pgroonga (content pgroonga_text_array_regexp_ops_v2);
```
Comment (kou) [inline sql/regexp/text-array/regexp-v2/empty-string/seqscan.sql:12]: これいる？

File: src/pgroonga.c [hunk 2]
```diff
@@ -4209,6 +4209,12 @@ pgroonga_regexp_text(PG_FUNCTION_ARGS)
 	PG_RETURN_BOOL(matched);
 }
 
+/**
+ * Caller must satisfy the following conditions:
+ * * targets is not empty
+ * * pattern is not NULL(TODO: We don't implement yet)
+ * * pattern is not an empty text
+ */
```
Comment (kou) [inline src/pgroonga.c:4217]: チェックする場所の変更もこのPRでやっちゃうの？

Comment (komainu8) [reply src/pgroonga.c:4217]: うーん。
空文字列のテストの追加なので、同時にチェックする場所の変更も入れたほうが収まりがいいかなと思ったのですが、このPRテストの追加なので、チェックする場所の変更は余計ですね。。。消します。

File: sql/regexp/text-array/regexp-v2/empty-string/seqscan.sql (hunk 1 above)
Comment (komainu8) [reply sql/regexp/text-array/regexp-v2/empty-string/seqscan.sql:12]: あぁ、シーケンシャルサーチなので、インデックスの設定いらないですね。消します。

File: sql/regexp/text-array/regexp-v2/empty-string/seqscan.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+);
+
+INSERT INTO memos
+     VALUES (ARRAY['PostgreSQL is a RDBMS',
+                   'Groonga is fast full text search engine',
+                   'PGroonga is a PostgreSQL extension that uses Groonga']);
+
+SET enable_seqscan = on;
+SET enable_indexscan = off;
+SET enable_bitmapscan = off;
```
Comment (komainu8) [inline sql/regexp/text-array/regexp-v2/empty-string/seqscan.sql:12]: インデックスがないから`SET`もなくてよい。

File: expected/regexp/text-array/regexp-v2/empty-string/seqscan.out [hunk 4]
```diff
@@ -0,0 +1,25 @@
+CREATE TABLE memos (
+  content text[]
```
Comment (kou) [inline expected/regexp/text-array/regexp-v2/empty-string/seqscan.out:2]: 配列だから`contexts`にしようか。

Comment (komainu8) [reply expected/regexp/text-array/regexp-v2/empty-string/seqscan.out]: おぉ。。。ありがとうございます。
`contents`にしました。
